lib/airbrake-ruby/response.rb

```diff
@@ -52,5 +63,10 @@ def self.truncated_body(body)
       end
     end
     private_class_method :truncated_body
+
+    def self.rate_limit_delay(response)
+      Time.now + response['X-RateLimit-Delay'].to_i
```

Comment:
Does this work when header is missing? It would be nice if it does.

Reply:
It works and equals to 0 because of type casting.